### PR TITLE
Realm SDK team feedback

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- Slack feedback

### Staged Changes (Requires MongoDB Corp SSO)

- [Kotlin SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/landingqsAPILinks/sdk/kotlin)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Comment by nathan-contino-mongo** [comment]
> Staging build complete! Visit it at: http://realm-io-staging.s3-website-us-east-1.amazonaws.com/nathan-contino-mongo/docs-realm/landingqsAPILinks/index.html

Note that this is a docusaurus staging link, seems to not reflect the changes in this PR properly right now (we just added this action on Friday). Check out the "staged changes" link in the PR description instead.

**Review by cbush — DISMISSED** [review]
Please don't merge with workflow changes

File: .github/workflows/docusaurus-staging.yml [hunk 1]
```diff
@@ -16,7 +16,9 @@ jobs:
       DOCUSAURUS_BASE_URL: "/${STAGING_DIR}/"
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```
Comment (mongodben) [inline .github/workflows/docusaurus-staging.yml:21]: what's this do?

**Review by mongodben — APPROVED** [review]
lgtm % chris comment abt workflow changes

**Comment by cbush** [comment]
The pull_request_target GitHub Action event always runs whatever version of the workflow file is on the upstream target branch. Therefore, changes to it in a fork PR _will never run_ until merged. So, it's better to try to fix the workflow on the upstream target branch first, then rebase your PR so the workflow can run again.

File: .github/workflows/docusaurus-staging.yml (hunk 1 above)
Comment (mongodben) [reply .github/workflows/docusaurus-staging.yml:21]: to make sure that it always gets the head of pr? https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit

Comment (nathan-contino-mongo) [reply .github/workflows/docusaurus-staging.yml]: That's the intent -- chatted with bush and it looks like we'll fix this elsewhere (so we can test this in this PR)

**Comment by cbush** [comment]
@mongodben meanwhile it would be interesting to see if we can get the bot to only post that comment once per PR

**Pull request closed without merging**